packages/doenetml/src/Viewer/DocViewer.tsx

```diff
@@ -698,13 +589,19 @@ export function DocViewer({
                 });
             }
             for (let componentIdx in args.rendererState) {
-                updateRendererSVsWithRecoil({
-                    coreId: coreId.current,
-                    componentIdx: Number(componentIdx),
-                    stateValues: args.rendererState[componentIdx].stateValues,
-                    childrenInstructions:
-                        args.rendererState[componentIdx].childrenInstructions,
-                });
+                dispatch(
+                    mainThunks.updateRendererSVs({
+                        coreId: coreId.current,
+                        componentIdx: Number(componentIdx),
```

Comment:
This will currently break given that `componentIdx` won't be turned into a number until the code that removes namespaces is merged! At this point, it is still a string so one should get `NaN` here. (This code is only called when loading in state from the database, so it's probably not being triggered when you test it.)

Reply:
I'll remove this and we'll just deal with the TS error for now.